### ISSUE TITLE
$previousState.get() returns null

### ISSUE THREAD

**Issue by teichmaj** [christopherthielen/ui-router-extras#166]
I am using a modal with a sticky state just as the example.
It works like a charm on my development server.
However, when I build my angular project with grunt it stops working as $previousState.get() returns null

Is there anything I have to do to make $previousState work using uglify, minify? I already annotate 
```
.state "admin.pwdChange",
     url: "/admin/pwd"
     template: "<div></div>"
     onEnter: ["$state", "$previousState", "PwdChange",
       ($state, $previousState, PwdChange) ->
         PwdChange().result.finally( () ->
            $previousState.go()
       )]
```

**Comment by JetFault** [comment]
One issue I can think of is, there might not be a previousState. This happens when you hit `/admin/pwd` directly (like on a page refresh).

Add a breakpoint whenever you call `$previousState.memo` and see what the current state is

**Comment by teichmaj** [comment]
I use the code successfully on the development server. There, everything is working perfectly. But when I use grunt to build my app it breaks. It is somehow related to the grunt uglify plugin.
Also setting the memo for the previous state returns null. 
Has anyone else used this plugin with uglify and minify?
My grunt setup is from the standard yo angular generator https://github.com/yeoman/generator-angular

**Comment by Tea56** [comment]
I have the same problem using grunt uglify
Unuglified code works just fine.

Using previousState, core and transition.

**Comment by Tea56** [comment]
Updating ng-annotate helped.

Using grunt ng-annotate 0.10.0 which resolves to ng-annotate 0.15.4 which is the latest today.
maybe add ng-annotate version requirement to the project?

**Comment by christopherthielen** [comment]
closing, since this is apparently an ng-annotate bug (thanks @Tea56)

I'm not going to add ng-annotate requirement, because the project doesn't require it.
